components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/importexport/ImportExportConstants.java

```diff
@@ -116,6 +122,10 @@ public final class ImportExportConstants {
     // Name of the API name element tag of the api.json file
     public static final String API_NAME_ELEMENT = "name";
 
+    public static final String OPERATION_POLICY_TYPE = "type";
+
+    public static final String DEFAULT_OPERATION_POLICY_TYPE = "operation_policy_specification";
```

Comment:
We already have this value under another constant. https://github.com/wso2/carbon-apimgt/blob/c3713fe9895975f1b6d869e22a3d34d08986720a/components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/importexport/ImportExportConstants.java#L208

Reply:
Fixed by [1bcdc33](https://github.com/wso2/carbon-apimgt/pull/11325/commits/1bcdc33fe9957d445f64c64ff88d33c30cf86456)